test-versions.txt

```diff
@@ -1,13 +1,20 @@
 # Well known working crio tag/commit/branch
 crio_version=959aab4fd5dbc315ca2dc84a90599c2108756a2c
 
+# Clear Containers image version
+image_version=17270
+
 # Kernel Version to use on recent Linux Distros
-kernel_clear_release=16680
-kernel_version="4.9.35-76"
+kernel_clear_release=17580
+kernel_version="4.9.47-77"
 
 # Kernel Version to use for Ubuntu 14.04 (Semaphore Env)
-kernel_clear_release=12760
-kernel_version="4.5-50"
+old_kernel_clear_release=12760
+old_kernel_version="4.5-50"
```

Comment:
I am not sure about these vars, maybe could be more descriptive something like `ubuntu_kernel_clear_release` or maybe
`semaphore_kernel_clear_release`

Reply:
agree, changed to `semaphore_kernel_clear_release`